pyproject.toml

```diff
@@ -21,7 +21,7 @@ dependencies = [
     "pendulum",
     "psycopg2-binary>=2.9.10",
     "boto3",
-    "bluecore-models>=0.8.1",
+    "bluecore-models",
```

Comment:
I like that this is not explicitly pinned now since we are the ones actively developing on it.  I feel like this will help reduce in the future with rapid development (and helping reduce issues where the pin was not updated)